composer.json

```diff
@@ -18,12 +18,16 @@
   "require": {
     "php": ">=5.5.0",
     "composer/installers": "*",
-    "silverstripe/framework": "^4.0",
-    "silverstripe/reports": "^4.0",
-    "silverstripe/siteconfig": "^4.0"
+    "silverstripe/framework": "^4.0-alpha3",
```

Comment:
I've identified that once alpha3 is tagged, these builds won't install dev-master of framework anymore. There's a risk that cms builds will be stuck on old framework releases.

I suggest to change prefer-stable back to false, but leave it as true in the silverstripe/installer.